index.html

```diff
@@ -2,8 +2,48 @@
 <html lang="ru">
 <head>
     <meta charset="utf-8">
-    <title>Задача «Подписка на бложик»</title>
+    <title>Подписка на бложик</title>
+    <link rel="stylesheet" href="index.css">
 </head>
 <body>
+<form class="sub-form">
+    <h1>Хочу подписаться!</h1>
+    <p>После вашего согласия на подписку вам будут приходить уведомления на почту и телефон</p>
+    <div class="main">
```

Comment:
Можно добавить семантики
```suggestion
    <main class="main">
```